resources/announcements.py

```diff
@@ -0,0 +1,130 @@
+from flask_restful import Resource
+from werkzeug.exceptions import BadRequest
+from flask import request,jsonify,g
+from datetime import datetime
+from sqlalchemy.orm.exc import NoResultFound
+from common.json_schema import Announcement_Schema
+from common.utils import headers,is_logged_in,has_admin_privileges
+from common.utils import bad_request,unauthorised,forbidden,not_found,internal_server_error,unprocessable_entity
+
+class Announcements_RUD(Resource):
+    """
```

Comment:
change this so it's not in the function but above it. I think it makes it much more readable that way.